### ISSUE TITLE
davinci-resolve: .desktop file missing

### ISSUE THREAD

**Issue by davidak** [NixOS/nixpkgs#166900]
### Describe the bug

Follow up to https://github.com/NixOS/nixpkgs/pull/152113.

This might help: https://github.com/NixOS/nixpkgs/pull/152113#discussion_r795106901

### Steps To Reproduce
1. ...
2. ...
3. ...

### Notify maintainers
@jshcmpbll

**Comment by yuuyins** [comment]
try
```
postInstall = ''
install -Dm644 share/DaVinciResolve.desktop $out/share/applications/
install -Dm644 share/DaVinciControlPanelsSetup.desktop $out/share/applications/
install -Dm644 share/DaVinciResolveInstaller.desktop $out/share/applications/
install -Dm644 share/DaVinciResolveCaptureLogs.desktop $out/share/applications/
install -Dm644 share/blackmagicraw-player.desktop $out/share/applications/
'';
```

ref https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=davinci-resolve

**Comment by pbsds** [comment]
Fixed in #240549
